src/tests/conformance/handshake.rs

```diff
@@ -11,10 +16,18 @@ async fn handshake_when_node_receives_connection() {
     // crate::tools::synthetic_node::enable_tracing();
 
     // Start the Ripple node
-    let mut node = Node::start_with_peers(vec![]).await.unwrap();
+    let mut node = Node::start(
+        home::home_dir()
+            .expect("Can't find home directory")
+            .join(ZIGGURAT_CONFIG),
+        IpAddr::V4(Ipv4Addr::LOCALHOST),
+        vec![],
+    )
+    .await
+    .unwrap();
 
     // Start synthetic node.
-    let synth_node = SyntheticNode::new(&TestConfig::new().unwrap()).await;
+    let synth_node = SyntheticNode::new(&TestConfig::default()).await;
```

Comment:
```suggestion
    let synth_node = SyntheticNode::new(&Default::default()).await;
```
applies to all such calls

Reply:
Done